tapalcatl_server/server.go

```diff
@@ -381,14 +382,24 @@ func main() {
 	// set if we have s3 storage configured, and shared across all s3 sessions
 	var awsSession *session.Session
 
+	for _, sd := range hc.Storage {
+		t := sd.Type
+		switch t {
+		case "s3":
+		case "file":
```

Comment:
I'm used to case statements without a `break` falling through, so this looked wrong to me. But I was wrong: https://tour.golang.org/flowcontrol/9.